css-nav-1/Overview.bs

```diff
@@ -1348,56 +1381,95 @@ Note: the 'spatial-navigation-contain' property is <a>at-risk</a>.
 </div>
 
 <h3 id=css-property-spatialnavigationaction>
-Defining the spatial navigation behavior : the 'spatial-navigation-action' property</h3>
+Controlling the interaction with scrolling: the 'spatial-navigation-action' property</h3>
 
 <pre class='propdef'>
 Name: spatial-navigation-action
 Value: auto | focus | scroll
 Initial: auto
-Applies to: <a>scroll container</a>
-Inherited: true
+Applies to: <a>scroll containers</a>
+Inherited: no
 </pre>
 
-When the user triggers spatial navigation in a given direction, the 'spatial-navigation-action' decides
-the proper spatial navigation behavior.
+When the focus is inside of a scroll container and the user triggers spatial navigation,
+it is somewhat ambiguous whether they are requesting that the focus be moved in that direction,
+or whether the document should be scrolled in that direction.
+By default, this is automatically determined,
+but this property allows the author decide between focusing or scrolling.
```

Comment:
"allows the author decide" -> "allows the author to decide"

Reply:
thanks